metricbeat/module/munin/node/node.go

```diff
@@ -0,0 +1,60 @@
+package node
```

Comment:
You wrote above that metricbeat is the master in this scenario so I was tempted to say we should call the default namespace `master`. But as it's about the data from the node (is that naming correct?) +1 on node.

Reply:
Protocol-wise we are like implementing the server side, but the metrics on the metricset are about the node, so +1 too to node :)